lib/css/modules/_userTagger.scss

```diff
@@ -111,6 +111,11 @@ a.voteWeight {
 
 .RESUserTag {
 	color: black;
+
+	// unselectable in comments/markdown for usability
+	.md & {
```

Comment:
Yeah https://github.com/erikdesjardins/Reddit-Enhancement-Suite/commit/2eadf20aea566c3e49204a888bf33f234b6a8c2c#diff-a34ff6560c6da1e0b6480724e857032aL2026

Unless you meant album captions/images, in which case I agree.